.changeset/wet-lamps-happen.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-search-backend-module-techdocs': minor
+'@backstage/plugin-search-backend-module-catalog': minor
+'@backstage/plugin-search-backend-module-explore': minor
```

Comment:
These are all new packages, just to confirm - it should still be minor changes and not patch?

Reply:
Just to clarify - I'm not sure. Would love @Rugvip's or any of he core maintainers input here.

Reply:
We usually start out with new packages as 0.0.0 in package.json and a minor level changeset, so the first released version becomes 0.1.0. So minor is good. But we also don't worry all _that_ much about that either; it ultimately doesn't matter super much in the end.

Reply:
Got it - @camilaibs lets keep minor here then! Thanks @freben

Reply:
Done!